packages/hash/frontend/src/components/hooks/useGetAccountIdForShortname.ts

```diff
@@ -14,7 +19,7 @@ export const useGetAccountIdForShortname = (
     const userBaseId = users?.find((user) => user.shortname === shortname)
       ?.entityEditionId.baseId;
     const userAccountId = userBaseId
-      ? extractEntityUuidFromEntityId(userBaseId)
+      ? extractAccountId(brand(userBaseId))
```

Comment:
Yep, the more I see this the more I think we'd want to see it as `extractAccountId(userBaseId as AccountEntityId)` 

But I don't know how we'd enforce that if both way works :(

Reply:
Changed in 79a98adc0227c976094cc723a4b84e1362ffa60d